ui/src/testing/stubs/notebooks-api-stub.ts

```diff
@@ -0,0 +1,17 @@
+import {NotebooksApi} from 'notebooks-generated/fetch';
+
+export class NotebooksApiStub extends NotebooksApi {
+
+  constructor() {
+    super(undefined, undefined, (..._: any[]) => {
+      throw Error('cannot fetch in tests');
+    });
+  }
+
+  public setCookie(googleProject: string, clusterName: string,
```

Comment:
nit: I'd put the arguments one per line, as it's odd to see the top line extend further than the next one.